src/briefcase/commands/create.py

```diff
@@ -36,6 +36,17 @@ def __init__(self, filename):
         )
 
 
+class InvalidSupportPackageURL(BriefcaseCommandError):
+    def __init__(self, url):
+        self.url = url
+        super().__init__(
+            "Invalid support package URL {url}; Compile the support package yourself, "
+            "add a support_package reference to pyproject.toml file".format(
+                url=url
+            )
+        )
```

Comment:
This is a user-facing error, so it's really important that it is both correct and actionable. In this case, the error could be caused: 

 * requesting a Python version that isn't supported
 * requesting an architecture that isn't supported
 * an invalid *custom* support package URL

"Invalid support package URL" isn't a helpful message for the first two cases; the user hasn't specified anything, so they haven't specified anything that is "invalid". 
 
"Compile the support package yourself" isn't a helpful message if you've already specified a custom support package.

We need to be able to differentiate between case 3 and case 1 & 2.

Reply:
For the cases where the Python Version and/or the Architecture isn't supported, do we maintain a list of the supported Versions and Architecture that can be referenced? 

Suggested Messages per Case :
- **For the Invalid Custom URL**
_Invalid support package URL_
-  **For Unsupported Architecture and Version**
_Unsupported Architecture/Version, please compile a support package yourself and add a support_package reference to pyproject.toml file_

Reply:
We don't maintain in *internal* list of supported versions and architectures - a 404 from the support platform download is the canonical source of truth for the list of supported versions and architectures.

As for the messages: A good error message should have two parts - a description of how the problem manifested, and a suggested course of action. "Invalid" isn't a great term to use - *why* is invalid? Was it badly formatted? Did it contain characters that aren't permitted? Or (as in this case) did the requested resource not exist?

However, don't get too distracted by the specific text in this case - no matter what you write, I'll almost certainly tweak it, so it's not worth worrying about it too much. :-)

Reply:
Got it thanks